src/bun.js/bindings/NodeVM.cpp

```diff
@@ -1233,6 +1509,11 @@ void configureNodeVM(JSC::VM& vm, Zig::GlobalObject* globalObject)
         [](const JSC::LazyProperty<JSC::JSGlobalObject, Structure>::Initializer& init) {
             init.set(createNodeVMGlobalObjectStructure(init.vm));
         });
+
+    globalObject->m_cachedNodeVMSpecialSandboxStructure.initLater(
+        [](const JSC::LazyProperty<JSC::JSGlobalObject, Structure>::Initializer& init) {
+            init.set(NodeVMSpecialSandbox::createStructure(init.vm, init.owner, init.owner->objectPrototype())); // TODO(@heimskr): or maybe jsNull() for the prototype?
```

Comment:
there's `globalObject->nullPrototype()` if that's what you meant by this comment